controllers/datamover.go

```diff
@@ -165,6 +167,65 @@ func (r *DPAReconciler) ReconcileDataMoverController(log logr.Logger) (bool, err
 	return true, nil
 }
 
+func (r *DPAReconciler) ReconcileDataMoverVolumeOptions(log logr.Logger) (bool, error) {
+
+	// fetch latest DPA instance
+	dpa := oadpv1alpha1.DataProtectionApplication{}
+	if err := r.Get(r.Context, r.NamespacedName, &dpa); err != nil {
+		return false, err
+	}
+
+	// check configMap already exists
+	confMap, confMapExists, err := r.checkConfigMapExists()
+	if err != nil {
+		return false, err
+	}
+
+	// check for existing configMap but no data mover configs set
+	if !r.checkDataMoverVolumeOptions(&dpa) && confMapExists {
+
+		err := r.Delete(context.Background(), confMap, &client.DeleteOptions{})
+		if err != nil {
+			return false, err
+		}
+	}
+
+	// create configmap only if data mover is enabled and has config values
+	if r.checkIfDataMoverIsEnabled(&dpa) && r.checkDataMoverVolumeOptions(&dpa) {
+
+		// create configmap to pass values to data mover CRs
+		cm := corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      DataMoverConfigMapName,
+				Namespace: dpa.Namespace,
```

Comment:
I believe you need to add OADP Operator labels to the configMap or else the operator predicate would filter out the reconcile events and OADP operator will not reconcile on the changes related to this configMap